### ISSUE TITLE
Make some basic product decisions

### ISSUE THREAD

**Issue by josephtyler** [Joe-Alfaro/crypto_app#3]
We need to decide on how this tool should be used.

*My thoughts*

Using Django, there can be a web interface, but I would probably want to use a CLI tool first. Maybe we can start with an API that both the app and the cli tool can communicate with. If that sounds good we can start to discuss what the API should look like.

**Comment by Joe-Alfaro** [comment]
I agree with an API that is flexible, maybe even leaving a possibility for a mobile app. The CLI tool sounds like step one though
